graphx/src/main/scala/org/apache/spark/graphx/GraphOps.scala

```diff
@@ -362,12 +362,14 @@ class GraphOps[VD: ClassTag, ED: ClassTag](graph: Graph[VD, ED]) extends Seriali
   def pregel[A: ClassTag](
       initialMsg: A,
       maxIterations: Int = Int.MaxValue,
+      checkpointInterval: Int = 25,
```

Comment:
isn't it a breaking change to add a param into the middle of the list?
also, this should be documented above, right?